client/optimizing_test.go

```diff
@@ -81,6 +82,7 @@ func TestOptimizingGet(t *testing.T) {
 
 	waitForSpeedTest(t, oc, 10*time.Second)
 
+	oc.Get(context.Background(), 0)
```

Comment:
I think this extra Get here is causing the test to fail, no?

Reply:
yes, you're right, I'm not sure why that is there but I think it might have been a stray paste that was done before commit because the tests were passing when I originally did this!